cordova/config.xml

```diff
@@ -94,7 +96,5 @@
     <plugin name="cordova-plugin-globalization" spec="https://github.com/apache/cordova-plugin-globalization.git" />
     <plugin name="cordova-plugin-inappbrowser" spec="https://github.com/apache/cordova-plugin-inappbrowser.git" />
     <plugin name="cordova-plugin-dialogs" spec="https://github.com/apache/cordova-plugin-dialogs.git" />
-    <plugin name="phonegap-plugin-push" spec="~1.9.2">
-        <variable name="SENDER_ID" value="XXXXXXX" />
-    </plugin>
+    <plugin name="phonegap-plugin-push" spec="~2.1.2"/>
```

Comment:
How is the SENDER_ID now set?

Reply:
We've specified senderId in `cordova` service
https://github.com/crossroads/stock.goodcity/blob/master/app/services/cordova.js#L30